index.html

```diff
@@ -0,0 +1,183 @@
+        <div class="">
+          <img src="logo2.svg" class="footer-logo align-bottom" alt="">
+          <span class="align-bottom copy text-secondary">&copy; 2021 NEWSWEEK DIGITAL, LLC</span>
+        </div>
+      </div>
+
+      <div class="col-12 col-md-4 d-flex align-items-center justify-content-md-end">
+        <ul class="d-flex list-group list-group-horizontal list-unstyled align-middle social">
+          <li>
+            <a href="#" class="text-secondary">
+              <i class="fa fa-facebook-square fa-2x"></i>
+            </a>
+          </li>
+          <li>
+            <a href="#" class="text-secondary">
+              <i class="fa fa-twitter fa-2x"></i>
+            </a>
+          </li>
+          <li>
+            <a href="#" class="text-secondary">
+              <i class="fa fa-instagram fa-2x"></i>
+            </a>
+          </li>
+          <li>
+            <a href="#" class="text-secondary">
+              <i class="fa fa-tumblr fa-2x"></i>
+            </a>
+          </li>
+          <li>
+            <a href="#" class="text-secondary">
+              <i class="fa fa-linkedin fa-2x"></i>
+            </a>
+          </li>
+        </ul>
+      </div>
+    </div>
+
+    <nav>
+      <div class="d-flex">
+        <strong>Edition:</strong>
+        <ul class="d-flex flex-wrap list-unstyled link-list-footer padding-start">
+          <li class="border-end-sm"><a href="#" class="text-secondary">U.S. Edition</a></li>
+          <li class="border-end-sm"><a href="#" class="text-secondary">日本</a></li>
+          <li class="border-end-sm"><a href="#" class="text-secondary">Pakistan</a></li>
+          <li class="border-end-sm"><a href="#" class="text-secondary">Polska</a></li>
+          <li><a href="#" class="text-secondary">Romania</a></li>
+        </ul>
+      </div>
+
+      <ul class="d-flex flex-wrap list-unstyled link-list-footer">
+        <li class="border-end-sm"><a href="#" class="text-secondary">About us</a></li>
+        <li class="border-end-sm"><a href="#" class="text-secondary">Corrections</a></li>
+        <li class="border-end-sm"><a href="#" class="text-secondary">Contact us</a></li>
+        <li class="border-end-sm"><a href="#" class="text-secondary">Editorial Guidelines</a></li>
+        <li class="border-end-sm"><a href="#" class="text-secondary">Advertise</a></li>
+        <li class="border-end-sm"><a href="#" class="text-secondary">Copyright</a></li>
+        <li class="border-end-sm"><a href="#" class="text-secondary">Terms & Conditions</a></li>
+        <li class="border-end-sm"><a href="#" class="text-secondary">Privacy Policy</a></li>
+        <li class="border-end-sm"><a href="#" class="text-secondary">Cookie Policy</a></li>
+        <li class="border-end-sm"><a href="#" class="text-secondary">Terms of Sale</a></li>
+        <li class="border-end-sm"><a href="#" class="text-secondary">Archive</a></li>
+        <li><a href="#" class="text-secondary">Announcements</a></li>
+      </ul>
```

Comment:
Hint: you can add bootstrap padding to each ul li items to closely match the original
**Original**
![Screenshot 2021-04-06 at 11 00 45](https://user-images.githubusercontent.com/57832450/113699117-74fefb00-96d5-11eb-8284-86330520800f.png)

**Yours**
![Screenshot 2021-04-06 at 12 40 55](https://user-images.githubusercontent.com/57832450/113699139-7d573600-96d5-11eb-9d46-8a7154dcf3c5.png)

Reply:
Alright, we are on it!